libraries/plugins/json_rpc/include/sophiatx/plugins/json_rpc/json_rpc_plugin.hpp

```diff
@@ -90,7 +90,15 @@ namespace detail
 class json_rpc_plugin : public appbase::plugin< json_rpc_plugin >
 {
    public:
-      json_rpc_plugin();
+      static std::shared_ptr<json_rpc_plugin> &get_plugin() {
+         static std::shared_ptr<json_rpc_plugin> instance(new json_rpc_plugin);
```

Comment:
use std::make_shared
https://stackoverflow.com/questions/18301511/stdshared-ptr-initialization-make-sharedfoo-vs-shared-ptrtnew-foo

Reply:
This will not work, since constructor `json_rpc_plugin` is private, i.e. inaccessible from make_shared